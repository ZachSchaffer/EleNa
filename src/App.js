import React from 'react';
import logo from './logo.svg';
import './App.css';

class App extends React.Component{
  render() {
    return (
        <div className="App">
          <header className="App-header">
            <img src={logo} className="App-logo" alt="logo" />
            <p>
              Caleb stinks.
            </p>
          </header>
        </div>
    );
  }
}

export default App;
