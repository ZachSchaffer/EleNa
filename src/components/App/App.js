import React from 'react';
import './App.css';
import Navbar from '../Navbar/Navbar';
import Routes from '../Routes/Routes';

export default class App extends React.Component {
  render() {
    return (
      <div>
        <Navbar></Navbar>
        <Routes></Routes>
      </div>
    );
  }
}
