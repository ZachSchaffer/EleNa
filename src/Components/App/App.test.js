import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

test('test App', () => {
    const tree = renderer.create(<App></App>).toJSON();
    expect(tree).toMatchSnapshot();
});
