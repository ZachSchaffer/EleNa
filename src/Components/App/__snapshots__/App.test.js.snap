// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test App 1`] = `
<div>
  <div>
    <header
      className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <p
          className="MuiTypography-root MuiTypography-body1"
        >
          EleNa: Team Gone Fishering
        </p>
      </div>
    </header>
  </div>
</div>
`;
